src/bin/cql-stress-scylla-bench/workload/timeseries_read.rs

```diff
@@ -28,7 +28,8 @@ struct TimeseriesRead {
 
 #[derive(Clone)]
 pub struct TimeseriesReadConfig {
-    pub partition_offset: i64,
+    // FIXME: why is this not used??
+    pub _partition_offset: i64,
     pub pks_per_generation: u64,
     pub cks_per_pk: u64,
     pub start_nanos: u64,
```

Comment:
Perhaps it's worth immediate investigation, not to let it fall into oblivion of numerous FIXMEs?

Reply:
@muzarski please don't forget about this.